client/packages/coldchain/src/Equipment/ListView/AddFromScannerButton.tsx

```diff
@@ -34,7 +34,8 @@ export const AddFromScannerButtonComponent = () => {
   const equipmentRoute = RouteBuilder.create(AppRoute.Coldchain).addPart(
     AppRoute.Equipment
   );
-  const { mutateAsync: scanAsset } = useAssets.document.scan();
+  const { mutateAsync: fetchAsset } = useAssets.document.fetch();
+  const { mutateAsync: fetchGS1 } = useAssets.document.gs1();
```

Comment:
NIT: can we call it `fetchAssetByGS1` here too? This feels like it should fetch GS1 data... maybe even `fetchOrCreateFromGS1`? 🤷‍♀️